src/backend/embedding/chunkfile_Weaviateprocess/chunkfile_deleteWeaviate.py

```diff
@@ -0,0 +1,44 @@
+import logging
+from weaviate import Client
+from src.backend.weaviate.weaviate_manager import WeaviateManager  # Import the updated WeaviateManager class
+
+# Configure logging
+logging.basicConfig(level=logging.DEBUG, format="%(asctime)s - %(levelname)s - %(message)s")
+logger = logging.getLogger(__name__)
+
+class DataDeletionManager:
+    def __init__(self):
+        try:
+            self.client = Client("http://localhost:8080")
+            logger.info("Successfully connected to Weaviate")
+            self.weaviate=WeaviateManager()
+        except Exception as e:
+            logger.error(f"Failed to initialize Weaviate connection: {e}")
+            raise e
+
+    def delete_objects_by_customer_and_filename(self,customer_guid, filename):
```

Comment:
This function should be a part of weaviate manager and we can delete this file chunkfile_deleteWeaviate.py